backend/src/question-service/routes/questions.ts

```diff
@@ -13,6 +13,6 @@ router.get("/categories", QuestionsController.getQuestionCategories);
 
 router.get("/:questionId", QuestionsController.getQuestion);
 
-// TODO: "/:questionId/update"
+router.put("/:questionId/update", QuestionsController.updateQuestion);
```

Comment:
should be `router.patch("/:questionId", QuestionsController.updateQuestion);`

Reply:
Addressed in next commit.